File: final-pjt-back/community/urls.py
from django.urls import path
from . import views

app_name = 'community'

urlpatterns = [
    path('article/', views.article_list, name="article_list"),
    path('article/<int:article_pk>/', views.article_detail, name="article_detail"),
    path('article/create_article/', views.create_article, name="create_article"),
    path('article/<int:article_pk>/like/', views.like_article, name="like_article"),
    #comment_list
    path('article/<int:article_pk>/comments/', views.create_comment, name="create_comment"),
    path('article/<int:article_pk>/comments/comment_list/', views.comment_list, name="comment_list"),
    path('article/<int:article_pk>/comments/<int:comment_pk>/', views.comment_detail, name="comment_detail"),
    path('article/<int:article_pk>/comments/<int:comment_pk>/like/', views.like_comment, name="like_comment" ),
    # vote
    path('article/<int:article_pk>/vote/', views.vote),
]

File: final-pjt-front/src/api/urls.js
const HOST = 'http://localhost:8000/api/v1/'

const ACCOUNTS = 'accounts/'
// const COMMUNITY = 'community/'
const MOVIES = 'movies/'
const ARTICLE = 'community/article/'

export default {
  accounts: {
    login: () => HOST + ACCOUNTS + 'login/',
    logout: () => HOST + ACCOUNTS + 'logout/',
    signup: () => HOST + ACCOUNTS + 'signup/',
    currentUserInfo: () => HOST + ACCOUNTS + 'user/',
    profile: username => HOST + ACCOUNTS + 'profile/' + username + '/',
    myArticles: username => HOST + ACCOUNTS + 'profile/' + username + '/my_articles/',
    myComments: username => HOST + ACCOUNTS + 'profile/' + username + '/my_comments/',
  },
  articles: {
    articleList: () => HOST + ARTICLE,
    articleDetail: articleId => HOST + ARTICLE + articleId + '/',
    articleNew: () => HOST + ARTICLE + 'create_article/',
    articleUpdate: articleId => HOST + ARTICLE + articleId +'/',
    articleLike: articleId => HOST + ARTICLE + articleId + '/like/',
    commentNew: articleId => HOST + ARTICLE + articleId + '/comments/',
    commentList: articleId => HOST + ARTICLE + articleId + '/comments/comment_list/',
    commentDetail: (articleId, commentId) => HOST + ARTICLE + articleId + '/comments/' + commentId + '/',
    commentLike: (articleId, commentId) => HOST + ARTICLE + articleId + '/comments/' + commentId + '/like/',
    voteArticle: (articleId) => HOST + ARTICLE + articleId + '/vote/'
  },
  movies: {
    movieList: () => HOST + MOVIES,
    movieRecommendation: () => HOST + MOVIES + 'recommend/',
    searchResult: (movieId) => HOST + MOVIES + movieId + '/search/',
  },
}